docs/guidance/build/system_architectures.md

```diff
@@ -49,33 +49,33 @@ The [releases and records](../../getting_started/releases_and_records) guidance
 
 ![Pull and Convert](../../_static/png/pullAndConvert.png)
 
-The key benefit of this approach is that the middleware system can store the change history. This is especially good when source systems do not maintain historic data.
+The key benefit of this approach is that the middleware system can store the change history. This is especially good when data sources do not maintain historic data.
 
-This approach might involve some changes to live systems to allow the middleware to pull data. The middleware system merges and centralizes data in a single place.
+This approach might involve some changes to data sources to allow the middleware to pull data. The middleware system merges and centralizes data in a single place.
 
-To add more source systems, the OCDS conversion module needs to be updated to pull data from the new system(s).
+To add more data sources, the OCDS conversion module needs to be updated to pull data from the new source(s).
 
 An important decision in the implementation is the frequency to pull data. If the frequency is low, there is a risk of losing the detail of individual changes.
 
-An alternative to the pull mechanism is to use a push mechanism in each source system. Specific events or changes to the data would trigger a data push to the middleware. This approach can mitigate the risk of losing individual changes. But this might involve bigger modifications to the source system(s).
+An alternative to the pull mechanism is to use a push mechanism in each data source. Specific events or changes to the data would trigger a data push to the middleware. This approach can mitigate the risk of losing individual changes. But this might involve bigger modifications to the data source(s).
 
 European Dynamics developed an e-procurement system with a similar approach for OCDS output. The system was built for the Zambian Public Procurement Agency.
 
 ### Convert and push
 
-This scenario is a combination of the two previous scenarios. Live systems perform the conversion of data to OCDS format. They push converted data to a middleware, which maintains an OCDS format datastore. An API in the middleware system serves the OCDS data.
+This scenario is a combination of the two previous scenarios. Data sources perform the conversion of data to OCDS format. They push converted data to a middleware, which maintains an OCDS format datastore. An API in the middleware system serves the OCDS data.
 
 ![Convert and Push](../../_static/png/convertAndPush.png)
 
-This approach puts the burden of data conversion in live systems. Yet it might be a solution for publishers with a single source system which does not store the change history.
+This approach puts the burden of data conversion in data sources. Yet it might be a solution for publishers with a single data source which does not store the change history.
```

Comment:
```suggestion
This approach puts the burden of data conversion on data sources. Yet it might be a solution for publishers with a single data source which does not store the change history.
```